.ci/azure-pipelines.yml

```
@@ -16,44 +16,61 @@ jobs:
   - job: build
     displayName: 'Build'
 
-    dependsOn: lint
-    condition: succeeded()
-
     pool:
       vmImage: 'ubuntu-latest'
 
     strategy:
       matrix:
-        bundle:
-          BuildConfiguration: Bundle
-        standalone:
-          BuildConfiguration: Standalone
-      maxParallel: 2
+        Development:
+          BuildConfiguration: development
+        Production:
+          BuildConfiguration: production
+        Standalone:
+          BuildConfiguration: standalone
+      maxParallel: 3
 
     steps:
     - task: NodeTool@0
       displayName: 'Install Node'
       inputs:
         versionSpec: '10.x'
 
-    - script: 'yarn install'
+    - task: Cache@2
+      displayName: 'Check Cache'
+      inputs:
+        key: 'yarn | yarn.lock'
+        path: 'node_modules'
+        cacheHitVar: CACHE_RESTORED
+
+    - script: 'yarn install --pure-lockfile'
```

```suggestion
    - script: 'yarn install --frozen-lockfile'
```

Otherwise, the lockfile is essentially useless...